wandb/integration/sb3/__init__.py

```diff
@@ -0,0 +1,106 @@
+"""W&B callback for sb3
+
+Really simple callback to get logging for each tree
+
+Example usage:
+
+```python
+import gym
+import time
+from stable_baselines3 import PPO
+from stable_baselines3.common.vec_env import DummyVecEnv
+from wandb.integration.sb3 import WandbCallback
+import wandb
+
+config = {
+    "policy_type": "MlpPolicy",
+    "total_timesteps": 25000
+}
+experiment_name = f"PPO_{int(time.time())}"
+
+def make_env():
+    env = gym.make("CartPole-v1")
+    env = gym.wrappers.Monitor(env, f"videos/{experiment_name}", force=True) # record videos
+    env = gym.wrappers.RecordEpisodeStatistics(env) # record stats such as returns
+    return env
+
+wandb.init(
+    name=experiment_name,
+    project="sb3",
+    config=config,
+    allow_val_change=True, # auto-populate config with the model's hyperparams
+    sync_tensorboard=True, # auto-upload metrics
+    monitor_gym=True,  # auto-upload the videos of agents playing the game
+    save_code=True,  # optional
+)
+
+env = DummyVecEnv([make_env])
+model = PPO(config["policy_type"], env, verbose=1, tensorboard_log=f"runs/{experiment_name}")
+model.learn(total_timesteps=config["total_timesteps"],
+    callback=WandbCallback(
+        gradient_save_freq=1000, # 1000 is a good frequency for both gradient logging
+        model_save_freq=1000,    # and model saving
+        model_save_path=f"models/{experiment_name}",
+))
+```
+"""
+
+import os
+
+from stable_baselines3.common.callbacks import BaseCallback
+import wandb
+
+
+class WandbCallback(BaseCallback):
+    """ Log SB3 experiments to Weights and Biases
+
+        - Added model tracking and uploading
```

Comment:
Maybe add to the doc string here, just to make it super clear to the user:

```
- `wandb.init(...)` must be called before the WandbCallback can be used
```

Reply:
done